test/unit/google-font-loader.test.ts

```diff
@@ -297,6 +309,7 @@ describe('@next/font/google loader', () => {
 
     test('Setting axes on non variable font', async () => {
       await expect(
+        // @ts-expect-error
```

Comment:
Should these be needed or do the types need updating?

Reply:
Not needed, fixed.